bitnami/airflow/templates/worker-statefulsets.yaml

```diff
@@ -0,0 +1,306 @@
+      app.kubernetes.io/instance: {{ $.Release.Name }}
+      app.kubernetes.io/component: worker
+  serviceName: {{ template "airflow.fullname" $ }}-headless
+  replicas: {{ .replicas }}
+  updateStrategy:
+    type: {{ $.Values.updateStrategy | quote }}
+    {{- if (eq "Recreate" $.Values.updateStrategy) }}
+    rollingUpdate: null
+    {{- else if $.Values.rollingUpdatePartition }}
+    rollingUpdate:
+      partition: {{ $.Values.rollingUpdatePartition }}
+    {{- end }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "airflow.name" $ }}
+        app.kubernetes.io/instance: {{ $.Release.Name }}
+        app.kubernetes.io/component: worker
+    spec:
+{{- include "airflow.imagePullSecrets" $ | indent 6 }}
+      {{- if $.Values.securityContext.enabled }}
+      securityContext:
+        fsGroup: {{ $.Values.securityContext.fsGroup }}
+        runAsUser: {{ $.Values.securityContext.runAsUser }}
+      {{- end }}
+      {{- if $.Values.nodeSelector }}
+      nodeSelector: {{ toYaml $.Values.nodeSelector | nindent 8 }}
+      {{- end }}
+      {{- if $.Values.tolerations }}
+      tolerations: {{ toYaml $.Values.tolerations | nindent 8 }}
+      {{- end }}
+      {{- if $.Values.affinity }}
+      affinity: {{ toYaml $.Values.affinity | nindent 8 }}
+      {{- end }}
+      {{- if or ($.Values.airflow.cloneDagFilesFromGit.enabled) (.additionalRepos.enabled) }}
+      initContainers:
+        {{- if $.Values.airflow.cloneDagFilesFromGit.enabled }}
+        - name: git-clone-dag-repository
+          image: "{{ template "git.image" $ }}"
+          imagePullPolicy: {{ $.Values.git.pullPolicy | quote }}
+          command:
+            - /bin/bash
+            - -ec
+            - |
+              git clone {{ $.Values.airflow.cloneDagFilesFromGit.repository }} --branch {{ $.Values.airflow.cloneDagFilesFromGit.branch }} /dags
+          volumeMounts:
+            - name: git-cloned-dag-files
+              mountPath: /dags
+            {{- if $.Values.airflow.cloneDagFilesFromGit.sshKeySecretName }}
+            - name: dag-git-keys
+              mountPath: /root/.ssh
+            {{- end }}
+        {{- end }}
+        {{- if .additionalRepos.enabled }}
+        {{- range .additionalRepos.repos }}
+        - name: git-clone-{{ .name }}
+          image: "{{ template "git.image" $ }}"
+          imagePullPolicy: {{ $.Values.git.pullPolicy | quote }}
+          command:
+            - /bin/bash
+            - -ec
+            - |
+              git clone {{ .repository }} --branch {{ .branch }} /{{ .name }}
+          volumeMounts:
+            - name: git-cloned-{{ .name }}
+              mountPath: /{{ .name }}
+            {{- if .sshKeySecretName }}
+            - name: {{ .name }}-git-keys
+              mountPath: /root/.ssh
+            {{- end }}
+        {{- end }}
+        {{- end }}
+      containers:
+        {{- if $.Values.airflow.cloneDagFilesFromGit.enabled }}
+        - name: dag-repo-syncer
```

Comment:
Now that I see this, and taking into account that it is a major change. I would add an option for not allowing the syncing of the repository. Something like `syncFilesFromGit`

Reply:
Not done yet